### PR TITLE
[new release] pretty_expressive (0.5)

### PR DESCRIPTION
**Pull request by sorawee**
A pretty expressive printer

- Project page: <a href="https://github.com/sorawee/pretty-expressive-ocaml">https://github.com/sorawee/pretty-expressive-ocaml</a>
- Documentation: <a href="https://sorawee.github.io/pretty-expressive-ocaml/">https://sorawee.github.io/pretty-expressive-ocaml/</a>

##### CHANGES:

* Improve performance in `two_columns` via the zipper data structure
  and caching.
* Fix a bug caused from a wrong renderer is used due to memoization.
  Thanks to @zbyrn who reported the issue (sorawee/pretty-expressive-ocaml#2)!

### CODE REVIEW

File: packages/pretty_expressive/pretty_expressive.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+maintainer: ["Sorawee Porncharoenwase <sorawee.pwase@gmail.com>"]
+authors: ["Sorawee Porncharoenwase <sorawee.pwase@gmail.com>"]
+license: "ISC"
+homepage: "https://github.com/sorawee/pretty-expressive-ocaml"
+doc: "https://sorawee.github.io/pretty-expressive-ocaml/"
+bug-reports: "https://github.com/sorawee/pretty-expressive-ocaml/issues"
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" {>= "3.10"}
+  "mdx" {>= "2.3.0" & <= "2.3.1" & with-test}
```
Comment (mseri) [inline packages/pretty_expressive/pretty_expressive.0.5/opam:14]: Is this still needed?

Comment (sorawee) [reply packages/pretty_expressive/pretty_expressive.0.5/opam:14]: #25476 is just merged. So I guess `(& (>= 2.3.0) (<> 2.4.0))` could also be used. Is it worth making the change?

Comment (sorawee) [reply packages/pretty_expressive/pretty_expressive.0.5/opam]: I made the aforementioned change

**Comment by mseri** [comment]
Thanks

**Pull request merged**